models/public/cocosnet/cocosnet-correspondence/model.yml

```diff
@@ -0,0 +1,29 @@
+# Copyright (c) 2020 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+description: >-
+  The purpose of correspondence model is to establish correspondence between input image and given exemplar.
+  Correspondence network return warped exemplar with semantic from input.
+task_type: image_translation
+files:
+  - name: corr.onnx
+    size: 237471853
+    sha256: cf2de274232bb9f6ad2acfe84c16aea71b196efd4f260a664ed000a090657f0d
+    source:
+      $type: google_drive
+      id: 1T8g6Lbyl1vRzPByN7dvdf694sSaCbij7
```

Comment:
Where did this file come from? I don't see any links to it in the original repository.

Reply:
Now it's a link to my own google drive. To get onnx format you need change symbolic_onnx in torch, so I skip the stage conversion from pytorch to onnx and provide onnx files.